templates/settings.html

```diff
@@ -4,6 +4,15 @@
 
 <h3>Settings</h3>
 
+<h4>Opt-out of Leaderboard</h4>
+<form method="get">
+  {% csrf_token %}
+  <div class="form-group">
+    <input class="form-control" id="opt-out" type="checkbox" name="opt-out">
+  </div>
+</form>
+<br>
```

Comment:
Instead of the br, just add a `class="mt-3"` to the form below this one :).

(or mt-2, whichever one looks nicer to you)